### ISSUE TITLE
Dashboard: include redis clients over time panel

### ISSUE THREAD

**Issue by naseemkullah** [oliver006/redis_exporter#284]
Hi, I was wondering if we could include a panel with clients over time? We found this to be useful in our org to detect increase spikes and so on.

The panels JSON would be something like (for v1.x.x):

```
{
  "aliasColors": {},
  "bars": false,
  "dashLength": 10,
  "dashes": false,
  "datasource": "Prometheus",
  "fill": 1,
  "gridPos": {
    "h": 8,
    "w": 12,
    "x": 0,
    "y": 28
  },
  "id": 16,
  "legend": {
    "avg": false,
    "current": false,
    "max": false,
    "min": false,
    "show": true,
    "total": false,
    "values": false
  },
  "lines": true,
  "linewidth": 1,
  "links": [],
  "nullPointMode": "null",
  "options": {},
  "percentage": false,
  "pointradius": 2,
  "points": false,
  "renderer": "flot",
  "seriesOverrides": [],
  "spaceLength": 10,
  "stack": false,
  "steppedLine": false,
  "targets": [
    {
      "expr": "redis_connected_clients{instance=~\"$instance\"}",
      "format": "time_series",
      "intervalFactor": 1,
      "refId": "A"
    }
  ],
  "thresholds": [],
  "timeFrom": null,
  "timeRegions": [],
  "timeShift": null,
  "title": "Clients",
  "tooltip": {
    "shared": true,
    "sort": 0,
    "value_type": "individual"
  },
  "type": "graph",
  "xaxis": {
    "buckets": null,
    "mode": "time",
    "name": null,
    "show": true,
    "values": []
  },
  "yaxes": [
    {
      "format": "short",
      "label": null,
      "logBase": 1,
      "max": null,
      "min": null,
      "show": true
    },
    {
      "format": "short",
      "label": null,
      "logBase": 1,
      "max": null,
      "min": null,
      "show": true
    }
  ],
  "yaxis": {
    "align": false,
    "alignLevel": null
  }
}
```

**Comment by naseemkullah** [comment]
(Not bug rather a feature request)

**Comment by oliver006** [comment]
Good suggestion, seeing # of clients over time can be a very handy when diagnosing/detecting issues.

I'm open for a PR to the Grafana dashboard, not sure how much time I'll have over the next couple of days/weeks to add this myself.

PS: I love the Transit app, using it every day :-)

**Comment by naseemkullah** [comment]
> Good suggestion, seeing # of clients over time can be a very handy when diagnosing/detecting issues.
> 
> I'm open for a PR to the Grafana dashboard, not sure how much time I'll have over the next couple of days/weeks to add this myself.

I can submit the PR in the near future!

> 
> PS: I love the Transit app, using it every day :-)

Great to hear! We love and use prometheus redis exporter every day! :)

**Comment by naseemkullah** [comment]
closed via #287
